go.mod

```diff
@@ -43,3 +43,5 @@ require (
 )
 
 replace github.com/sirupsen/logrus => github.com/sirupsen/logrus v1.4.2
+
+replace github.com/containers/common => github.com/containers/common v0.43.3-0.20210830085322-6143ffe54e5f
```

Comment:
Please remove the `replace` line in `go.mod` and do a `go get github.com/containers/common@6143ffe54e5f`.

The replace lines would otherwise prevent dependabot from bumping to new releases.

Reply:
ah my bad sure.